spec/requests/healthcheck_metrics_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe "/healthcheck/metrics" do
+  it "returns distinct organisations ordered by title" do
+    get "/healthcheck/metrics"
+    json = JSON.parse(response.body)
+
+    expect(json["checks"]).to include("daily_metrics")
+    .and(include("etl_metric_values_pviews"))
+    .and(include("etl_metric_values_searches"))
+    .and(include("etl_metric_values_upviews"))
+    .and(include("etl_metric_values_feedex"))
+  end
+
+  it "is not cacheable" do
+    get "/healthcheck"
```

Comment:
```suggestion
    get "/healthcheck/metrics"
```

But I wonder if it's worth having a `before { get "/healthcheck/metrics" }` in the `describe` block? It might need refactoring later if we want a custom request, but for these tests it should work.

Reply:
Good idea.  I've moved these into a `before`.